WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteSourceManager.kt

```diff
@@ -0,0 +1,123 @@
+package org.wordpress.android.ui.mysite
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.distinctUntilChanged
+import kotlinx.coroutines.CoroutineScope
+import org.wordpress.android.analytics.AnalyticsTracker.Stat
+import org.wordpress.android.ui.mysite.MySiteSource.MySiteRefreshSource
+import org.wordpress.android.ui.mysite.MySiteSource.SiteIndependentSource
+import org.wordpress.android.ui.mysite.MySiteUiState.PartialState
+import org.wordpress.android.ui.mysite.cards.domainregistration.DomainRegistrationSource
+import org.wordpress.android.ui.mysite.cards.post.PostCardsSource
+import org.wordpress.android.ui.mysite.cards.quickstart.QuickStartCardSource
+import org.wordpress.android.ui.mysite.dynamiccards.DynamicCardMenuViewModel.DynamicCardMenuInteraction
+import org.wordpress.android.ui.mysite.dynamiccards.DynamicCardMenuViewModel.DynamicCardMenuInteraction.Hide
+import org.wordpress.android.ui.mysite.dynamiccards.DynamicCardMenuViewModel.DynamicCardMenuInteraction.Pin
+import org.wordpress.android.ui.mysite.dynamiccards.DynamicCardMenuViewModel.DynamicCardMenuInteraction.Unpin
+import org.wordpress.android.ui.mysite.dynamiccards.DynamicCardsSource
+import org.wordpress.android.util.analytics.AnalyticsTrackerWrapper
+import org.wordpress.android.util.config.MySiteDashboardPhase2FeatureConfig
+import javax.inject.Inject
+
+@Suppress("LongParameterList")
```

Comment:
Minor (🔍): This suppression is no longer required, consider removing it.

Reply:
🤔  @ParaskP7  this shows in AS as a detekt problem ... am I missing something?
![Screen Shot 2021-11-29 at 9 38 52 AM](https://user-images.githubusercontent.com/506707/143887486-8ee1d4bd-2140-48dc-bb7d-b3e037103516.png)

Reply:
Hmmm... I run `WordPress:detekt` after removing that and it wasn't showing a problem, I wonder what is happening... 🤔 Can you run this task on CLI and let me know if you get a warning as well? Maybe the IDE got out-of-sync somehow.

Reply:
If I remove it and run `./gradlew WordPress:detekt` I don't get a warning, but AS give me the warning. So very odd.

Reply:
I guess I can ignore AS and just the CLI, but dang, it's so nice to see the warnings while coding. :(

Reply:
removed in 364fca1

Reply:
> I guess I can ignore AS and just the CLI, but dang, it's so nice to see the warnings while coding. :(

Hey, yes, it is super nice and helpful, please don't ignore the warnings in general in AS, maybe this one was a weird one, as I wasn't seeing in it my AS. Maybe it is some kind of integration/sync problem, we can sit together to double check it if you like!